internal/controllers/import_controller.go

```diff
@@ -39,6 +39,7 @@ import (
 
 const (
 	importAnnotation             = "rancher-auto-import"
```

Comment:
I think this should be removed to satisfy the fact that the labels should be used instead of annotation

Reply:
I am not sure, Labels usually are used by Kubernetes control plane services, annotation are much more dedicated for third party operators.

Reply:
I have checked and Rancher is mostly using annotations, which is right approach